### DISCUSSION TITLE
Apktool has started putting decompiled files in weird locations

### DISCUSSION THREAD

**Discussion by SalamiSimon** [iBotPeaches/Apktool#3406] [General]
Recently decompiled resources has appeared in my downloads folder, but it's duplicate files since they're in the default ApkTool_M folder too (that I have set as output location).

It happened once so I thought it was a one time thing so I moved the files I wanted to save to my Documents folder and deleted my whole downloads folder, was like 40k files, took a while. 

Then now I noticed it had happened again, but not only in my Downloads folder but my Documents folder too! Not only that, it says the folder is 5GB and it's lagging when I enter the folder (File by Google app) but I get "something went wrong" when I try to delete them. If I use another file browser like Root browser or FX the folder is empty and with no lag at all. 

Really weird. I've tried clearing cache on the apps (though I'm not sure if I can clear system cache on my Nothing phone), ran CCleaner and restarted my phone. 

Any ideas?

**Comment by iBotPeaches** [comment]
May you give exact command you use? Also the current working directory so I can see if spaces or pound signs.

**Reply by SalamiSimon** [reply]
Oh I'm sorry I think I made a mistake, I thought this was the GitHub page for ApkTool M by Maximoff. 

Sorry about that.
